src/components/select/Select.stories.tsx

```diff
@@ -0,0 +1,148 @@
+                 defaultHelperText="You have these choices"
+                 errorHelperText="This field is required!"
+                 label="I've got some options"
+             >
+                 <SelectOption value="1">One</SelectOption>
+                 <SelectOption value="2">Two</SelectOption>
+                 <SelectOption value="3">Three</SelectOption>
+             </Select>
+ 
+             <Select
+                 error={true}
+                 onBlur={action("select with error - blur")}
+                 onChange={action("select with error - change")}
+                 defaultHelperText="You have these choices"
+                 errorHelperText="This field is required!"
+                 label="I've got some options"
+             >
+                 <SelectOption value="1">One</SelectOption>
+                 <SelectOption value="2">Two</SelectOption>
+                 <SelectOption value="3">Three</SelectOption>
+             </Select>
+         </div>
+     ))
+     .add("Disabled select", () => (
+         <div>
+             <Select onChange={action("Disabled select - change")} disabled={true} label="I've got some options">
+                 <SelectOption value="1">One</SelectOption>
+                 <SelectOption value="2">Two</SelectOption>
+                 <SelectOption value="3">Three</SelectOption>
+             </Select>
+         </div>
+     ))
+     .add("Select Option Group", () => (
+         <div>
+             <Select onChange={action("Option group select - change")}>
+                 <SelectOptionGroup label="Group 1">
+                     <SelectOption value="1">One</SelectOption>
+                     <SelectOption value="2">Two</SelectOption>
+                     <SelectOption value="3">Three</SelectOption>
+                 </SelectOptionGroup>
+                 <SelectOptionGroup label="Group 2">
+                     <SelectOption value="7">Seven</SelectOption>
+                     <SelectOption value="8">Eight</SelectOption>
+                     <SelectOption value="9">Nine</SelectOption>
+                 </SelectOptionGroup>
+             </Select>
+         </div>
+     ))
+     .add("Select Option Group Disabled", () => (
+         <div>
+             <Select onChange={action("Option Group select Disabled - change")}>
+                 <SelectOptionGroup label="Group 1" disabled>
+                     <SelectOption value="1">One</SelectOption>
+                     <SelectOption value="2">Two</SelectOption>
+                     <SelectOption value="3">Three</SelectOption>
+                 </SelectOptionGroup>
+                 <SelectOptionGroup label="Group 2">
+                     <SelectOption value="7">Seven</SelectOption>
+                     <SelectOption value="8">Eight</SelectOption>
+                     <SelectOption value="9">Nine</SelectOption>
+                 </SelectOptionGroup>
+             </Select>
+         </div>
+     ));
+ 
```

Comment:
Please add empty line at end of file

Reply:
there is an empty line at the end of the file. I am not sure what exactly this means